From 2d743eb94a3b28d365c1524ab4a7427809688d10 Mon Sep 17 00:00:00 2001
From: Abdullah Yousufi <abdullah@cloudera.com>
Date: Wed, 6 Jul 2016 09:49:08 -0500
Subject: [PATCH 0659/1164] CDH-42033: HIVE-14149: Joda Time causes an
 AmazonS3Exception on Hadoop3.0.0 (Abdullah
 Yousufi, reviewed by Sergio Pena)

Change-Id: I597cf532d7463379ec324103db988997b7bda204
---
 pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pom.xml b/pom.xml
index f65bfd9..01a32b0 100644
--- a/pom.xml
+++ b/pom.xml
@@ -148,7 +148,7 @@
     <jersey.version>1.14</jersey.version>
     <jline.version>2.12</jline.version>
     <jms.version>1.1</jms.version>
-    <joda.version>2.5</joda.version>
+    <joda.version>2.8.1</joda.version>
     <jodd.version>3.5.2</jodd.version>
     <json.version>20090211</json.version>
     <jsp.api.version>2.1</jsp.api.version>
-- 
1.7.9.5

